tests/test_elastic_search_visitor.py

```diff
@@ -2600,3 +2600,52 @@ def test_nested_query_exact_last_name():
 
     generated_es_query = _parse_query(query_string)
     assert generated_es_query == expected_es_query
+
+
+def test_elastic_search_visitor_find_journal_with_year():
+    query_str = "j jhep,0903,112,2009"
```

Comment:
this should work with
```suggestion
    query_str = "j jhep,0903,112"
```

Reply:
But how can it get the year if it wouldn't be passed? Should the field with publication year be null in query then?

Reply:
```